onnxruntime/core/optimizer/gelu_fusion.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/optimizer/graph_transformer.h"
+
+namespace onnxruntime {
+
+class GeluFusion : public GraphTransformer {
+ public:
```

Comment:
A comment describing what this class fuses would be helpful. Shouldn't have to read the implementation to figure out details like which nodes are involved and when they're fused.